api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -306,6 +310,10 @@ public void deleteWorkspace(DbWorkspace dbWorkspace) {
     dbWorkspace.setWorkspaceActiveStatusEnum(WorkspaceActiveStatus.DELETED);
     dbWorkspace = saveWithLastModified(dbWorkspace);
     maybeDeleteRecentWorkspace(dbWorkspace.getWorkspaceId());
+
+    String billingProjectName = dbWorkspace.getWorkspaceNamespace();
```

Comment:
Have you thought through the transactionality of errors here? If we succeed in deleting the workspace, but fail to delete the project, are we happy?

Reply:
Yes - that would be equivalent to the current behavior.  We will always be in a world where we need to go back and clean up orphaned projects (due to local DB wipes) so this is acceptable.

I'll update this to catch-and-log errors